android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -435,6 +435,10 @@ protected void onResume() {
 
         final SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
 
+        id = getIntent().getIntExtra(ConstantStrings.ID, 0);
```

Comment:
Getting this value from Extras creating problem in onCreate() method?
We are already  getting id in the onCreate() method. 
https://github.com/rohanagarwal94/open-event-android/blob/ec9f4ab95d166fddcd31b559b61b6adc376079ac/android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java#L145